UIComponets/InstagramProfileScreen/ViewControllers/InstagramProfileVC.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  InstagramProfileVC.swift
+//  UIComponets
+//
+//  Created by Rajan Patel on 02/06/23.
+//
+
+import UIKit
+
+class InstagramProfileVC: UIViewController {
```

Comment:
why we have not added BaseViewController?, deinit is also not added, how we have check memory leak for this class